test/makeLatexReportRegression.m

```diff
@@ -0,0 +1,231 @@
+function makeLatexReportRegression(currentStatus, output, otherStatus)
+% generate a LaTeX report
+%
+% 
+    if ~exist('output','var')
+        output = m2troot('test','output','current');
+    end
+    % first, initialize the tex output
+    SM = StreamMaker();
+    stream = SM.make(fullfile(output, 'regression.tex'), 'w');
+
+    texfile_init(stream);
+
+    printFigures(stream, currentStatus, otherStatus);
+    printSummaryTable(stream, currentStatus);
```

Comment:
I think it's a bit weird to only show detailed stuff (summary, error messages, env info) for the current status. Wouldn't it make sense to show it also for the other (old?) situation? E.g. to see when we fix an unreliable test, introduce a new one, ...?

I think eventually we could do something similar to those parts as what you have done with the figures; but in the meanwhile just both overviews separately would be more than sufficient.